generators/entity-client/templates/vue/src/main/webapp/app/entities/entity.component.ts.ejs

```diff
@@ -24,7 +24,7 @@ export default class <%= entityAngularName %> extends <% if (fieldsContainBlob |
   public page = 1;
   public previousPage = 1;
   public propOrder = 'id';
-  public reverse = <%_ if (pagination === 'infinite-scroll') { _%>true<%_ } else { _%>false<%_ } _%>;
+  public reverse = false;
```

Comment:
why does this condition is removed?

Reply:
because by default entities must be sorted by id ASC and not reverted

Reply:
I'm still wonder why do we set it to true when choosing the infinite-scroll

Reply:
it was a mistake because with true, entities are bad sorted